### ISSUE TITLE
question about CCN

### ISSUE THREAD

**Issue by luotianhang** [menyifang/DCT-Net#47]
i have made a styleganv2 and trained it on ffhq for 80w iters with bs == 32 , i think the pretrain model is enought. And i am trying to build the ccn u mentioned in the paper. I found that ur paper doesn't talk about the weight of  gan loss and id loss, can u show the part of ccn in details thank u
